backend/src/middlewares/userValidator.js

```diff
@@ -0,0 +1,34 @@
+import { check, validationResult } from 'express-validator';
+
+export const validateUser = [
```

Comment:
@marcianszlovak create unit tests for middleware to check the validation results